### PR TITLE
fix(ci): properly name overview as index.md, as required by mkdocs.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/workflows/pages.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         run: |
           mkdir docs/
           pip install -r rules_inventory/scripts/requirements.txt
-          python rules_inventory/scripts/rules_overview_generator.py --rules_file=rules/falco_rules.yaml > docs/overview.md
+          python rules_inventory/scripts/rules_overview_generator.py --rules_file=rules/falco_rules.yaml > docs/index.md
```
Comment (FedeDP) [inline .github/workflows/pages.yaml:32]: As per mkdocs documentation, default page must be named either `index.md` or `README.md`

**Review by incertum — DISMISSED** [review]
/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**